Training/interface/DataLoader_main.h

```diff
@@ -192,7 +205,7 @@ class DataLoader {
       if (!target_th2d) throw std::runtime_error("Target histogram could not be loaded");
 
       for( auto const& [tau_type, tau_name] : tau_types_names)
-      {
+      { if (tau_type != 8){ // No histogram for data samples
```

Comment:
- fix indentations 
- use enum instead of 8 (also below)

Reply:
I've now avoided using 8 and match to "data", by using tau_name for the first case and the unordered map find for the second, assuming this is ok